axelor-base/src/main/resources/domains/MailBatch.xml

```diff
@@ -8,7 +8,7 @@
   <entity name="MailBatch" lang="java">
   	
   	<!-- HEADER -->
-  	<string name="code" title="Code" namecolumn="true" unique="true"/>
+    <string name="code" title="Code" namecolumn="true" unique="true" required="true"/>
```

Comment:
Code should be able to be null if the batch is run from UI. Only required if the batch is called per WS from a scheduler.